traffic_ops/app/lib/UI/Server.pm

```
@@ -375,10 +375,13 @@ sub check_server_input {
 	if ( defined( $paramHashRef->{'ip6_address'} )
```

Can you please check if the user enters in an ip6_gateway.  Currently, if a user enters an ip6_gateway and no ip6_address the ui just silently ignores the value.